src/index.ts

```diff
@@ -15,6 +15,9 @@ export { model } from "amf-client-js";
 
 export * from "./generate";
 
+// eslint-disable-next-line prettier/prettier
```

Comment:
Would upgrading to a newer version of prettier make it not complain?

Reply:
I didn't try.. I will try that in the followup PR